src/index.js

```diff
@@ -81,67 +79,83 @@ function clickLog(event) {
 
 function validateForm() {
     let validated = true;
-    // let validated = false;
     const selectedDate = document.querySelector('.input-date');
     const selectedDuration = document.querySelector('.input-duration');
     const selectedTravelers = document.querySelector('.input-travelers');
     const selectedDestination = document.querySelector('.input-destination');
     const dateError = document.querySelector('.date-error');
     const durationError = document.querySelector('.duration-error');
     const travelersError = document.querySelector('.travelers-error');
-    // let destinationError = document.querySelector('.destination-error');
     dateError.classList.add('hidden');
     durationError.classList.add('hidden');
     travelersError.classList.add('hidden');
-    // destinationError.classList.add('hidden');
     if (!moment(selectedDate.value)._isValid || moment(selectedDate.value).isBefore(moment(Date.now()))) {
         dateError.classList.remove('hidden');
         validated = false;
-    // } else {
-    //     validated = true;
     }
     if (!selectedDuration.value || typeof(+selectedDuration.value) !== 'number') {
```

Comment:
```suggestion
    if (!selectedDuration.value || isNaN(selectedDuration.value)) {
```


I don't think this will work if someone passes in a string.
To check if something is a number we should use isNaN (is Not a Number)
```
let selectedDuration = {value: "hello"}
> undefined

selectedDuration
> {value: "hello"}

selectedDuration.value
> "hello"

!selectedDuration.value || typeof(+selectedDuration.value) !== 'number'
> false (incorrect)

!selectedDuration.value || isNaN(selectedDuration.value)
> true (correct)
```

Reply:
Why would someone do that!!!